frontend/packages/policy-editor/src/components/PolicyEditor.tsx

```diff
@@ -105,7 +109,7 @@ export const PolicyEditor = ({
           subjects={subjects}
           rules={policyRules}
           setPolicyRules={setPolicyRules}
-          resourceId={resourceId}
+          resourceId={resourceId ?? ''}
```

Comment:
resourceId is not nullable, then we could use it as a key instead of using the index. Using index as key is not recommended by React.

_You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a key at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs._ - https://react.dev/learn/rendering-lists#why-does-react-need-keys

Reply:
It's coming from the props, and it can be undefined. The reason is that the for resources, there is a resource id, but when you use the policyEditor for apps there is no resource id. 
![Screenshot 2023-08-23 at 10 02 06](https://github.com/Altinn/altinn-studio/assets/133344438/81c8bb7b-9871-4f65-bfbd-9a98e00a0ff8)

I'll update the ID as requested :)